### ISSUE TITLE
Sonar CSS raising false positives for angular material css styles

### ISSUE THREAD

**Issue by pkumarch** [SonarSource/sonar-css#204]
Sonar is not able to recognize CSS styles definition for angularJS material design components. 
![image](https://user-images.githubusercontent.com/34448209/64612165-c4455280-d3f0-11e9-8d4a-de2dcda00128.png)

This is similar to issue #151 .

**Comment by vilchik-elena** [comment]
Hi, 

We will add this pattern to default value of rule parameter. Still note that you can already change that parameter to avoid these FPs.

**Comment by NagatoHvirtual** [comment]
This rule say that you should not use tag in css, you can use css class and this is a example:

HTML
```
<md-card class="card">
  <md-card-content class="padding">
    Hello World!
  </md-card-content>
</md-card>
```

CSS
```
.card .padding{
  padding: 8px;
}
```

Test: https://codepen.io/nagatohvirtual/pen/qBENYrZ

**Comment by thw0rted** [comment]
I'd like to direct anybody watching this issue to [this Community post](https://community.sonarsource.com/t/stop-flagging-valid-custom-web-component-tag-names-with-s4670/30262) I just made which is relevant to your interests. Short version: stop checking tag names with dashes in them.
